utilities/joint-position-tracking/README.md

```diff
@@ -0,0 +1,27 @@
+## joint-position-tracking
+
+The **joint-position-tracking** is a simple tool for testing the joint position tracking of a robot in a `YARP` environment.
+
+
+
+# :running: How to use the application
+The fastest way to use the application is to run the `sh` script [`run_test.sh`](./script/run_test.sh).
```

Comment:
I would specify from which folder this script has to run

Reply:
Thanks to c4d4e391f31638bb9e2a9b59ea362cff84e4540c the script can ran in any folder. I also updated the README in 16f0c92b975ad38c566fb3899b04b58bd57c5099 

Thanks @traversaro for the help